dhall/src/Dhall/Import.hs

```diff
@@ -792,12 +793,25 @@ getCacheDirectory = alternative₀ <|> alternative₁
             Just xdgCacheHome -> return xdgCacheHome
             Nothing           -> empty
 
-    alternative₁ = do
-        maybeHomeDirectory <- liftIO (System.Environment.lookupEnv "HOME")
+    alternative₁
+        | isWindows = do
+            maybeLocalAppDirectory <-
+              liftIO (System.Environment.lookupEnv "LOCALAPPDATA")
+
+            case maybeLocalAppDirectory of
+                Just localAppDirectory -> return localAppDirectory
+                Nothing                -> empty
+
+        | otherwise = do
+            maybeHomeDirectory <-
+              liftIO (System.Environment.lookupEnv "HOME")
+
+            case maybeHomeDirectory of
+                Just homeDirectory -> return (homeDirectory </> ".cache")
+                Nothing            -> empty
+
+        where isWindows = System.Info.os == "mingw32"
```

Comment:
I wish we wouldn't have to hardcode this magic string ourselves! :/

Reply:
Yeah, i didn't find a standard `Bool` function for this, see
* https://gitlab.haskell.org/ghc/ghc/issues/1590
* but it is http://hackage.haskell.org/package/extra-1.6.18/docs/System-Info-Extra.html#v:isWindows
  * if we are already importing it we could use but i would not add the dep only for this

Reply:
Ah, thanks for the links! I guess this is the pragmatic solution then! :+1:

Reply:
Oh, in fact we have it as transitive dependency (I don't know why but it doesn't surprise me :smile:) so i think we dont pay much for adding it as explicit one, right?

Reply:
I can't find `extra` with `stack dot dhall --external`.

Reply:
i was just executing dot and looking for it with no luck but `cabal build dhall` says:
```
    Could not load module ‘System.Info.Extra’
    It is a member of the hidden package ‘extra-1.6.18’.
    Perhaps you need to add ‘extra’ to the build-depends in your .cabal file.file.
```
:thinking:

Reply:
Never mind i misinterpreted "hidden package" :worried:

Reply:
No worries! Let's see what @Gabriel439 thinks and finally get this merged! :rocket:

Reply:
The cause of my misinterpretation is that `cabal v1-build` had a different error for modules not listed in dependencies:
```
src\Dhall\Import.hs:205:1: error:
    Could not find module `System.Info.Extra'
    Use -v to see a list of the files searched for.
    |
205 | import qualified System.Info.Extra
    | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

```
It seems that if the package is in the v2 store it sees it as `hidden package` (???)
Anyway sorry for the noise.